src/app/ApplicationTemplate.Access/ApiClients/Posts/PostRepositoryException.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace ApplicationTemplate.DataAccess;
+
+public class PostRepositoryException : Exception
```

Comment:
I would seal it but it's your choice!

Reply:
I would do this for every renamed class but only if you have 15 minutes to spend on it.

Reply:
it's too unrelated to the changes. I only renamed stuff (and removed a few unused classes instead of renaming them).